src/sentry/rules/conditions/existing_high_priority_issue.py

```diff
@@ -33,19 +33,24 @@ def get_activity(
                 project=self.project,
                 datetime__gte=start,
                 datetime__lt=end,
-                type__in=[ActivityType.SET_ESCALATING.value],
+                type__in=[ActivityType.SET_PRIORITY.value],
                 user_id=None,
             )
             .order_by("-datetime")[:limit]
             .values_list("group", "datetime", "data")
         )
 
+        activities = [
+            (group_id, timestamp, data)
+            for group_id, timestamp, data in activities
+            if data.get("priority") == "high"
+        ]
+
         return [
             ConditionActivity(
                 group_id=group_id,
-                type=ConditionActivityType.REAPPEARED,
+                type=ConditionActivityType.EXISTING_HIGH_PRIORITY_ISSUE,
```

Comment:
why is this changing?

Reply:
REAPPEARED is used for escalating errors, but this condition has more nuance so I've added two new ConditionActivityType values.